modules/oc-compliance-fetching-raw-results.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * security/oc_compliance_plug_in/oc-compliance-plug-in-using.adoc
+
+[id="fetching-raw-results_{context}"]
+= Fetching raw results
+
+When a compliance scan finishes, the results of the individual checks are listed in the resulting `ComplianceCheckResult` custom resource (CR). However, an administrator or auditor might require the complete details of the scan. The OpenSCAP tool creates an Advanced Recording Format (ARF) formatted file with the detailed results. This ARF file is too large to store in a config map or other standard Kubernetes resource, so a persistent volume (PV) is created to contain it.
+
+.Procedure
+
+* Fetching the results from the PV with the Compliance Operator is a four-step process. However, with the `oc` Compliance Plug-in, you can use a single command:
++
+[source,terminal]
+----
+$ oc compliance fetch-raw <object-type> <object-name> -o <output-path>
+----
++
+* `<object-type>` can be either `scansettingbinding`, `compliancescan` or `compliancesuite`, depending on which of these objects the scans were launched with.
+* `<object-name>` is the name of the binding, suite, or scan object to gather the ARF file for, and `<output-path>` is the local directory to place the results.
++
+For example:
++
+[source.terminal]
+----
+$ oc compliance fetch-raw scansettingbindings my-binding -o /tmp/
+----
++
+.Example output
+[source.terminal]
+----
+Fetching results for my-binding scans: ocp4-cis, ocp4-cis-node-worker, ocp4-cis-node-master
+Fetching raw compliance results for scan 'ocp4-cis'.......
+The raw compliance results are avaliable in the following directory: /tmp/ocp4-cis
+Fetching raw compliance results for scan 'ocp4-cis-node-worker'...........
+The raw compliance results are avaliable in the following directory: /tmp/ocp4-cis-node-worker
+Fetching raw compliance results for scan 'ocp4-cis-node-master'......
+The raw compliance results are avaliable in the following directory: /tmp/ocp4-cis-node-master
+$ ls /tmp/ocp4-cis-node-master/
+ocp4-cis-node-master-ip-10-0-128-89.ec2.internal-pod.xml.bzip2  ocp4-cis-node-master-ip-10-0-150-5.ec2.internal-pod.xml.bzip2  ocp4-cis-node-master-ip-10-0-163-32.ec2.internal-pod.xml.bzip2
+----
```

Comment:
@ahardin-rh Hi, it will be helpful if we add following `bunzip2` command to extract and view result.
For example :

> $ bunzip2 -c resultsdir/worker-scan/worker-scan-stage-459-tqkg7-compute-0-pod.xml.bzip2 > resultsdir/worker-scan/worker-scan-ip-10-0-170-231.us-east-2.compute.internal-pod.xml 
> 
> $ ls resultsdir/worker-scan/
> worker-scan-ip-10-0-170-231.us-east-2.compute.internal-pod.xml  
> worker-scan-stage-459-tqkg7-compute-0-pod.xml.bzip2            
> worker-scan-stage-459-tqkg7-compute-1-pod.xml.bzip2
> 
> $ less resultsdir/worker-scan/worker-scan-ip-10-0-170-231.us-east-2.compute.internal-pod.xml
>